heart_disease/src/data/make_dataset.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from typing import Tuple
+
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+from src.entities import SplittingParams
```

Comment:
Здесь можно использовать relative import
```suggestion
from ..entities import SplittingParams
```